homeassistant/components/envisalink/__init__.py

```diff
@@ -227,6 +235,13 @@ async def handle_custom_function(call: ServiceCall) -> None:
                 hass, "binary_sensor", "envisalink", {CONF_ZONES: zones}, config
             )
         )
+        """Only DSC panels support getting zone bypass status"""
+        if panel_type == "DSC":
```

Comment:
Consider making "DSC" a constant instead of hardcoding the string in this if statement. For example,
```PANEL_TYPE_SUPPORT_ZONE_BYPASS = "DSC"```

Reply:
Updated to use constants for the different panel types.